expression/expression.go

```diff
@@ -269,6 +275,30 @@ func TableInfo2Schema(tbl *model.TableInfo) Schema {
 		}
 		schema.Append(newCol)
 	}
+	for _, idx := range tbl.Indices {
+		if !idx.Unique || idx.State != model.StatePublic {
+			continue
+		}
+		newKey := make([]*Column, 0, len(idx.Columns))
+		for _, idxCol := range idx.Columns {
+			for i, col := range tbl.Columns {
```

Comment:
I notice that find the position of the column in used in several places,  can we also make a function like `FindColPos` in `table` package?